packages/base-subgraph/tsconfig.json

```diff
@@ -0,0 +1,70 @@
+[
```

Comment:
this looks like an ABI file, why is it a `tsconfig.json` fie?

Reply:
Good catch, likely the result of working with 5+ different windows open on a 14" screen. Fixed.